content/en/blog/_posts/2025-xx-xx-ippr-goes-ga.md

```diff
@@ -0,0 +1,123 @@
+* **Address transient resource needs** Workloads that temporarily need more resources can be adjusted quickly. This enables features like the CPU Startup Boost ([AEP-7862](https://github.com/kubernetes/autoscaler/pull/7863)) where applications can request more CPU during startup and then automatically scale back down.
+
+Here are a few examples of some use cases:
+
+* A game server that needs to adjust its size with shifting player count.
+* A pre-warmed worker that can be shrunk while unused but inflated with the first request.
+* Dynamically scale with load for efficient bin-packing.
+* Increased resources for JIT compilation on startup.
+
+## Changes between Beta (1.33) and Stable (1.35)
+
+Since the initial beta in v1.33, development effort has primarily been around stabilizing the feature and
+improving its usability based on community feedback. Here are the primary changes for the stable release:
+
+* **Memory limit decrease** Decreasing memory limits was previously prohibited. This restriction has been
+lifted, and memory limit decreases are now permitted. The Kubelet attempts to prevent OOM-kills by allowing the
+resize only if the current memory usage is below the new desired limit. However, this check is best-effort and
+not guaranteed.
+* **Prioritized resizes** If a node doesn't have enough room to accept all resize requests, _Deferred_ resizes 
+are reattempted based on the following priority:
+  * PriorityClass
+  * QoS class
+  * Duration _Deferred_, with older requests prioritized first.
+* **Pod Level Resources (Alpha)** Support for in-place Pod Resize with Pod Level Resources has been introduced 
+behind its own feature gate, which is alpha in v1.35.
+* **Increased observability**: There are now new Kubelet metrics and Pod events specifically associated with
+In-Place Pod Resize to help users track and debug resource changes.
+
+## What's next?
+
+The graduation of In-Place Pod Resize to stable opens the door for powerful integrations across the Kubernetes
+ecosystem. There are several areas for futher improvement that are currently planned.
+
+### Integration with autoscalers and other projects
+
+We hope to support integrations with several autoscalers and other projects to improve workload efficiency at a larger scale. Some projects under discussion:
+
+- VPA CPU startup boost ([AEP-7862](https://github.com/kubernetes/autoscaler/pull/7863)): Allows applications to request more CPU at startup and scale back down after a specific period of time.
+- VPA Support for in-place updates ([AEP-4016](https://github.com/kubernetes/autoscaler/tree/455d29039bf6b1eb9f784f498f28769a8698bc21/vertical-pod-autoscaler/enhancements/4016-in-place-updates-support)): VPA support for `InPlaceOrRecreate` has recently graduated to beta. As the feature stabilizes, we hope to see
+`InPlaceOrRecreate` graduate to stable. Support for `InPlaceOnly` mode is still under discussion.
```

Comment:
```suggestion
`InPlaceOrRecreate` graduate to stable. Support for `InPlace` mode is still under discussion.
```

This is being worked on in https://github.com/kubernetes/autoscaler/pull/8818, unsure if it's worth a mention?